# DP: Add "Internal compiler error:" before some errors
# DP:  by Franz Sirl

Index: gcc/toplev.c
--- toplev.c	1999/05/20 10:40:05	1.185.4.1
+++ gcc/toplev.c	1999/06/12 18:55:32
@@ -1467,6 +1467,8 @@ fatal_insn VPROTO((const char *msgid, rt
 #endif
   va_list ap;
 
+  error ("Internal compiler error:");
+
   VA_START (ap, insn);
 
 #ifndef ANSI_PROTOTYPES
@@ -1943,6 +1945,8 @@ fatal VPROTO((const char *msgid, ...))
   const char *msgid;
 #endif
   va_list ap;
+
+  error ("Internal compiler error:");
 
   VA_START (ap, msgid);
 
